inst/shiny/server.R

```diff
@@ -62,84 +66,84 @@ load_data <- function() {
 server <- shinyServer(function(input, output, session) {
   #For theming
   #bs_themer()
-    sessionid <- random_id(n = 1)
-    
+    sessionid <- digest(runif(10))
+
     #User event logging ----
-    if(file.exists(".db_url")){ #Should also allow people to disable these options.
+    if(db){ #Should also allow people to disable these options.
       set_logging_session()
-      log_event(sessionid)
-      observe(
-        log_value(input$intensity_corr)
-      )
-      observe(
-        log_value(digest::digest(preprocessed_data(), algo = "md5"))
-      )   
-      observe(
-        log_value(input$smoother)
-      )
-      observe(
-        log_value(input$baseline)
-      )
-      observe(
-        log_value(input$MinRange)
-      )
-      observe(
-        log_value(input$MaxRange)
-      )
-      observe(
-        log_value(input$event_rows_selected)
-      )
-      observe(
-        log_value(input$smooth_decision)
-      )
-      observe(
-        log_value(input$baseline_decision)
-      )
-      observe(
-        log_value(input$range_decision)
-      )
-      observe(
-        log_value(input$Spectra)
-      )
-      observe(
-        log_value(input$Data)
-      )
-      observe(
-        log_value(input$Library)
-      )
-      observe(
-        log_value(input$fingerprint)
-      )
-      observe(
-        log_value(input$ipid)
-      )
+          log_event(sessionid)
+          observe(
+            log_value(input$intensity_corr)
+          )
+          observe(
+            log_value(digest::digest(preprocessed_data(), algo = "md5"))
+          )
+          observe(
+            log_value(input$smoother)
+          )
+          observe(
+            log_value(input$baseline)
+          )
+          observe(
+            log_value(input$MinRange)
+          )
+          observe(
+            log_value(input$MaxRange)
+          )
+          observe(
+            log_value(input$event_rows_selected)
+          )
+          observe(
+            log_value(input$smooth_decision)
+          )
+          observe(
+            log_value(input$baseline_decision)
+          )
+          observe(
+            log_value(input$range_decision)
+          )
+          observe(
+            log_value(input$Spectra)
+          )
+          observe(
+            log_value(input$Data)
+          )
+          observe(
+            log_value(input$Library)
+          )
+          observe(
+            log_value(input$fingerprint)
+          )
+          observe(
+            log_value(input$ipid)
+          )
     }
-  
+
   # Loading overlay
   load_data()
   hide(id = "loading_overlay", anim = TRUE, animType = "fade")
   show("app_content")
 
-  # For desktop version of the app.
```

Comment:
Don't we need this anymore?

Reply:
No, I don't think we need that desktop code any more. That was just if we host Open Specy as a standalone application which I am not finding a ton of people using or requesting so I think we should just drop support for it for now. I think that code made it so that the app could be run using a bash script and wouldn't crash. We can always circle back if there is a request later and pick this back up. Let's leave it there hashed out in case we do want it so that we don't have to find it in our commits.